active/0000-remove-ember-cli-eslint.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2018-08-13
+- RFC PR: (leave this empty)
+
+# Summary
+
+Remove https://github.com/ember-cli/ember-cli-eslint from projects generated by
+`ember-cli`.
+
+[ember-cli-eslint](https://github.com/ember-cli/ember-cli-eslint) is an addon
+designed to show lint errors during test runs. Tooling around `eslint` has
+improved enough where this feature may no longer be necessary.
+
+# Motivation
+
+1. Improve our build speed
+2. Simplicity. `eslint` is common among JS stack, and integrations with editors
+   / precommit-hooks are ubiquitous. Removing this layer of abstraction will
+   simplify how `eslint` is used throughout `ember-cli`. Most editors have
+   plugins available for `eslint`, and as long as the `.eslint.rc` is not
+   removed, we should still see the benefits of `eslint` in our Ember projects.
+3. Hacks required to support features such as [PR #122
+   broccoli-lint-eslint](https://github.com/ember-cli/broccoli-lint-eslint/pull/122#discussion-diff-153937455R28)
+
+# Detailed design
+
+1. Change blueprint to pull in `eslint` as opposed to `ember-cli-eslint` under
+   `devDependencies`.
+2. Provide documentation on `eslint` and editor integration as well as precommit hooks
+
+
+# How We Teach This
+
+Providing documentation should suffice. Also, going towards a much more explicit
+path, this should become _easier_ to teach because of less abstractions.
+
+# Drawbacks
+
+1. No console warnings during builds
+2. lint failures are no longer included in browser tests
+
+# Alternatives
+
+N/A
```

Comment:
We at least mention the alternative of: "do nothing" (not that I think that is a good thing to do...)